fsharp-backend/tests/FuzzTests/Passwords.FuzzTests.fs

```diff
@@ -21,7 +21,7 @@ module RT = LibExecution.RuntimeTypes
 module G = Generators
 
 type Generator =
-  static member SafeString() : Arbitrary<string> =
+  static member String() : Arbitrary<string> =
```

Comment:
Why did you rename these from SafeString?

Reply:
I did it to prevent [this problem](https://github.com/darklang/dark/blob/4237ff6b227abe87c2580d0072377630633d368c/fsharp-backend/tests/FuzzTests/HttpClient.FuzzTests.fs#L35-L42). Which of these two generators will be used?

It's really easy to write conflicting generators unless we decide to consistently name the members to be of the type that we're overriding a generator for. Happy to just "keep this in mind" always if you think that's best, but this seemed like a decent way to prevent the problem.

Reply:
So what's the solution to the problem? Can you add a comment describing the problem and solution?

Reply:
The solution is consistent naming to prevent conflicting `static member`s.
One solution would be to update the coding guides - I've just pushed an adjustment there.

I haven't yet thought of a better solution - open to ideas

Reply:
Oh so the idea is name them all String so we don't accidentally have two types of string?

Reply:
yes, though I suppose that strategy fails a bit when we define generators for tuples - what's the naming convention then? One name followed by the other? (e.g. in ExecutePureFunctinos)